File: QuasarCode_Python/QuasarCode/Games/Dice/Dice12.py
from QuasarCode.Games.Dice import NDice

class Dice12(NDice):
    """
    Simple dice with 12 sides

    __init__():
        int seed --> seed for the random generator (deafult is None)
    """
    def __init__(self, seed = None):
        super.__init__(12, seed)

File: QuasarCode_Library_Games/Dice/Dice12.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QuasarCode.Library.Games.Dice
{
    /// <summary>
    /// Simple didce with 12 sides.
    /// </summary>
    public sealed class Dice12 : NDice
    {
        /// <summary>
        /// Creates a new Dice12 instance
        /// </summary>
        public Dice12() : base(12) { }

        /// <summary>
        /// Creates a new Dice12 instance with a seed for the random generator
        /// </summary>
        /// <param name="seed">Seed for the random generator</param>
        public Dice12(int seed) : base(12, seed) { }
    }
}
